beacon_node/lighthouse_network/src/types/pubsub.rs

```diff
@@ -270,6 +275,42 @@ impl<E: EthSpec> PubsubMessage<E> {
                             )),
                         }
                     }
+                    GossipKind::DataColumnSidecar(subnet_id) => {
+                        match fork_context.from_context_bytes(gossip_topic.fork_digest) {
```

Comment:
Use deneb_activated()

Reply:
Updated to use `deneb_enabled` in https://github.com/sigp/lighthouse/pull/6147/commits/f4cd1fa47c750a73b32760109e7a0ec91a1e0704